extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php

```diff
@@ -10,8 +10,14 @@ class ARecoveryEngineHooks {
 	 * @return bool
 	 */
 	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
+		global $wgDevelEnvironment;
 
 		if ( ARecoveryModule::isEnabled() ) {
+			$spStatusURL = 'http://community.wikia.com/api/v1/ARecoveryEngine/SPStatus';
```

Comment:
That's true. It's like our "analytics providers" or ComScore/Quantcast. Sorry for wasting time :(